### PR TITLE
Removing check on LSF jobs and adding echo of HTC queue setting.

### PR DESCRIPTION
**Pull request by bdalena**
Is this the PR you meant?

### CODE REVIEW

File: utilities/bash/run_status [hunk 1]
```diff
@@ -308,31 +308,31 @@ then
...
-           sixdeskexitparam=4
-           sixdeskexit
-#MACRO myexit
+##MACRO myexit 4
+#           sixdeskexitparam=4
+#           sixdeskexit
+##MACRO myexit
 
-        fi
+#        fi
```
Comment (amereghe) [inline utilities/bash/run_status]: I think you can delete these lines, and add an error message saying:"LSF no longer available"

**Review by amereghe — APPROVED** [review]

**Comment by amereghe** [comment]
Hi @bdalena ,
thanks a lot for the changes. Just a minor request.
Cheers,
A.

**Comment by bdalena** [comment]
Hi,
I think I will delete up to line 449 (i.e. all checks on LSF ) without adding error message concerning LSF. Maybe I can just leave a comment in the script that the check on LSF has been removed. What do you think?

**Comment by bdalena** [comment]
Hi,
I have just committed my last changes. I think I am done with them.
should I create another pull request or you can consider them and close this request?

ciao
Barbara

**Review by amereghe — APPROVED** [review]
Hi @bdalena ,
thanks for the PR, and sorry for approving it so late.
Cheers,

**Pull request merged**